### PR TITLE
adds handling for undefined external properties in local computed pro…

### PR DESCRIPTION
**Pull request by tschmits**
…perties

### CODE REVIEW

File: app/scripts/components/modelcard.js [hunk 1]
```diff
@@ -13,7 +13,10 @@ var exports = (function () {
 
     computed: {
       selectable: function () {
-        return (this.model.data.shared === "p");
+        if(_.has(this.model, "data.shared")) {
+          return (this.model.data.shared === "p");
+        }
+        return false;
```
Comment (SiggyF) [inline app/scripts/components/modelcard.js]: `return _.get(this.model, 'data.shared') === 'p'`

File: app/scripts/components/modeldetails.js [hunk 2]
```diff
@@ -90,7 +90,10 @@ var exports = (function () {
       delft3DVersion: {
         cache: false,
         get: function () {
-          return this.activeModel.data.versions.delft3d.delft3d_version;
+          if(_.has(this.activeModel, "data.versions.delft3d.delft3d_version")) {
+            return this.activeModel.data.versions.delft3d.delft3d_version;
```
Comment (SiggyF) [inline app/scripts/components/modeldetails.js]: perhaps make `delft3d`... a bit more generic (model.version, model.engine)

File: app/scripts/components/modelcard.js (hunk 1 above)
Comment (tschmits) [inline app/scripts/components/modelcard.js]: Thanks, Fedor, will change it.

File: app/scripts/components/modeldetails.js (hunk 2 above)
Comment (tschmits) [inline app/scripts/components/modeldetails.js]: I get what you mean, but this will incorporate many changes - I will create a task for it in Jira.

**Pull request merged**